python/src/trezorlib/cli/firmware.py

```diff
@@ -590,3 +590,12 @@ def update(
         click.echo("Dry run. Not uploading firmware to device.")
     else:
         upload_firmware_into_device(client=client, firmware_data=firmware_data)
+
+
+@cli.command()
+@click.argument("hex_challenge", required=False)
+@with_client
+def get_hash(client: "TrezorClient", hex_challenge: Optional[str]) -> str:
```

Comment:
going forward, we should extend this command with the ability to compare to an existing fw image